libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/gutenberg/GutenbergEditorFragment.java

```diff
@@ -544,6 +557,26 @@ public void onActivityResult(int requestCode, int resultCode, @Nullable Intent d
         return otherMediaOptions;
     }
 
+    private ArrayList<MediaOption> initOtherMediaAudioFileOptions() {
+        ArrayList<MediaOption> otherMediaOptions = new ArrayList<>();
+
+        FragmentActivity activity = getActivity();
+        if (activity == null) {
+            AppLog.e(T.EDITOR,
+                    "Failed to initialize other media options because the activity is null");
+            return otherMediaOptions;
+        }
+
+        String packageName = activity.getApplication().getPackageName();
+
+        int chooseAudioFileResourceId =
+                getResources().getIdentifier("photo_picker_choose_file", "string", packageName);
+
+        otherMediaOptions.add(new MediaOption(MEDIA_SOURCE_AUDIO_FILE, getString(chooseAudioFileResourceId)));
```

Comment:
When it comes to GB, GlueCode etc I'm still on the learning path 😄  , so not sure the following is worth or can have any constraint, but was wondering given the only difference seems to be the `MEDIA_SOURCE_FILE` vs `MEDIA_SOURCE_AUDIO_FILE` parameter, should we try to extract here also the common code with `initOtherMediaFileOptions`? wdyt?

Reply:
That sounds like a good idea @develric Thanks! I am going to look into doing this.

Reply:
Done e4039c0 c724657